generation/src/lib.rs

```diff
@@ -63,6 +65,8 @@ pub struct Field {
     pub strict_conversion: Option<TypePathOrEnum>,
     pub start: u32,
     pub end: Option<u32>,
+    #[serde(skip)]
+    pub attributes: Vec<syn::Attribute>,
```

Comment:
Let's make all of these purely cfg attrs and make the name reflect that just as with the register